html/pfappserver/lib/pfappserver/Base/Form/Role/WithCustomFields.pm

```diff
@@ -28,6 +28,18 @@ has_field 'custom_fields' =>
              help => 'The additionnal fields that should be required for registration' },
   );
 
+has_field 'fields_to_save' =>
+  (
+   type => 'Select',
+   multiple => 1,
+   label => 'Fields to save',
+   options_method => \&options_custom_fields,
+   element_class => ['chzn-select'],
+   element_attr => {'data-placeholder' => 'Click to add a required field'},
```

Comment:
Maybe just "Click to add a field" since its not part of the required fields per say